tests/js/app-tests.js

```diff
@@ -132,15 +132,18 @@ module.exports = {
             }
         };
         Realm.deleteFile(realmConfig);
-
+        console.log("KAT 1");
```

Comment:
these `console.logs` are debugging leftovers from another branch

Reply:
Yeah, that was the commit to ignore since I'm going to remove it during rebase 🙂

Reply:
👍